src/app/stack/card-details/card-details.component.html

```diff
@@ -5,9 +5,10 @@ <h2 [innerText]="details?.title"></h2>
         <p [innerText]="details?.titleDescription"></p>
     </span>
     <span class="col-sm-2" *ngIf="this.whatCard == 'security' || this.whatCard == 'compDetails'">
-        <a [href]="registrationLink" title="Sign up with Snyk" target="_blank">
+        <a [href]="registrationLink" title="Sign up with Snyk" target="_blank"
+            (click)="this.handleUrl.open($event,registrationLink)">
```

Comment:
```suggestion
            (click)="this.handleUrl.open($event, registrationLink)">
```

Reply:
Ya it is backward compatible, cmd + click behaviour will continue to exist.